src/types/events.d.ts

```diff
@@ -1,3 +1,19 @@
+import { DeviceType } from "./auctions";
+
+export type ChannelType = "onsite" | "offsite" | "instore";
+export type PageType = "home" | "category" | "PDP" | "search" | "cart" | "other";
```

Comment:
keep in mind that if page is cart then value becomes an array (list of products).

Also, can we add descriptions as documentation so the typing experience is better?